pkg/assets/applier.go

```diff
@@ -12,7 +12,7 @@ var (
 
 var assetsEventRecorder events.Recorder = events.NewLoggingEventRecorder("microshift-assets")
 
-type RenderParams map[string]string
+type RenderParams map[string]interface{}
```

Comment:
We talked about this offline, but for my recollection later: The parameters to the render function need to be fairly flexible to support different kubeadmin and cert files, so we don't want a full struct type here with a fixed set of fields. We may reexamine that decision in the future, but for now let's stick with the map.